components/toast/Toast.js

```diff
@@ -2,23 +2,26 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import Transition from 'react-transition-group/Transition';
 import cx from 'classnames';
-import { Button, IconButton } from '../button';
+import { IconButton } from '../button';
+import { TextSmall } from '../typography';
+import Link from '../link';
```

Comment:
I think Link is a bit misused here, since we want it to act like a button (so not an `<a>`). I think we need to create another component `LinkButton` which is actually a button, but looks like a link.

```js
const LinkButton = props => <Link element="button" data-teamleader-ui="link-button" {...props} />;
```